frontends/concrete-python/concrete/fhe/compilation/configuration.py

```diff
@@ -1053,6 +1055,8 @@ def __init__(
         optimize_tlu_based_on_measured_bounds: bool = False,
         enable_tlu_fusing: bool = True,
         print_tlu_fusing: bool = False,
+        optimize_tlu_based_on_original_bit_width: bool = True,
+        optimize_tlu_based_on_original_bit_width_threshold: int = 8,
```

Comment:
could be merged into 1 option ?
True = always
False = never
int = threshold

Reply:
Done!